src/components/hand-controls2.js

```diff
@@ -12,17 +12,33 @@ const POSES = {
   mrpDown: "mrpDown"
 };
 
-// TODO: If the hands or controllers are mispositioned, then rightHand.controllerPose and rightHand.pose
-//       should be bound differently.
-export const CONTROLLER_OFFSETS = {
+export const LEFT_CONTROLLER_OFFSETS = {
```

Comment:
Nice -- these offsets have been just slightly wrong for so long. Once we get off tracked-controls, these offsets can just be set on a per-device basis in the bindings so that the correct poses are stored in the frame at the `hand` paths